content/docs/command-reference/diff.md

```diff
@@ -44,18 +44,13 @@ for example when `dvc init` was used with the `--no-scm` option.
 
 ## Options
 
-- `--targets <paths>` - limit command scope to these paths. When specifying
-  arguments for `--targets` before `a_rev`/`b_rev`, you should use `--` after
-  this option's arguments, e.g.:
+- `--targets <paths>` - specific DVC-tracked files to compare.
 
-  ```dvc
-  $ dvc diff --targets t1.json t2.yaml -- HEAD v1
-  ```
-
-  Alternatively, you can also run the above statement as:
+  When specifying arguments for `--targets` before `a_rev`/`b_rev`, you should
+  use `--` after this option's arguments (POSIX terminals), e.g.:
 
   ```dvc
-  $ dvc diff HEAD v1 --targets t1.json t2.json
+  $ dvc diff --targets t1.json t2.yaml -- HEAD v1
   ```
```

Comment:
@shcheklein I have a couple emails about this change that I can't find in the PR. Do you still think this change complicates users' life? I was trying to simplify this actually, but I can revert, if you prefer.

Reply:
p.s. Recently you've asked me to keep other cmd ref. options short so I thought you would approve of this.

Reply:
I think I deleted them :) so we are good, I was confused by reading diff

Reply:
sorry for the confusion, I think it was fine as it was before, previous iteration. Up to you to revert.

Reply:
personally prefer the shorter version. The 2nd example is overkill for an option bullet and it's implied already.